index.bs

```diff
@@ -78,8 +78,8 @@ Ambient Light Sensor provides information about lighting conditions near
 the device environment. Potential privacy risks include:
 
   - Profiling. Ambient Light Sensor can leak information about user's use
-    patterns and surrounding. This information can be used to enhance user
-    profiling and behavioral analysis.
+    patterns and surrounding cross-origin. This information can be used to
```

Comment:
It's not about the language ;) Cross-origin sounds like we're speaking about web/origins. I'm speaking about surrounding, like, even the physical environment.